components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -1370,39 +1373,43 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
                   ! config_btr_gam3_velWt2=  0
                   !    flux = normalBarotropicVelocityOld*H
 
+                  !$omp parallel
+                  !$omp do schedule(runtime)
+                  do iCell = 1, nCells
+                     ! SSH is linear combination of SSHold and SSHnew
+                     sshTemp(iCell) = (1-config_btr_gam2_SSHWt1)* &
+                                            sshSubcycleCur(iCell) &
+                                    +    config_btr_gam2_SSHWt1 * &
+                                            sshSubcycleNew(iCell)
+                  end do ! cell loop for ssh
+                  !$omp end do
+                  !$omp end parallel
+
+                  !$omp parallel
+                  !$omp do schedule(runtime)
+                  do iEdge = 1, nEdgesAll
+                     btrvel_temp(iEdge) = (1.0-config_btr_gam3_velWt2) * &
+                                  normalBarotropicVelocitySubcycleCur(iEdge) &
+                                +       config_btr_gam3_velWt2 *             &
+                                  normalBarotropicVelocitySubcycleNew(iEdge)
```

Comment:
Again, I think this will be more readable if the indentation is like above -- `+` aligned with `=`.